src/ui/page/notFoundError/notFoundError.scss

```diff
@@ -0,0 +1,106 @@
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/mixins.scss';
+
+@mixin background-image($theme) {
```

Comment:
As a suggestion, I would have enhanced this function to be more generic and then to be used by any other component.

Reply:
@fredericvilcot I think that this function is too specific to be made generic and it might get too complicated to do that. Since we have agreed that after both error pages get merged, I will start working on refactoring and making one error component, I think this should be solved better then than now. What do you think?